Cargo.lock

```diff
@@ -117,6 +117,14 @@ name = "glob"
 version = "0.2.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
```

Comment:
Should we be checking in Cargo.lock?

Reply:
If you’re building a non-end product, such as a rust library that other rust packages will depend on, put Cargo.lock in your .gitignore. If you’re building an end product, which are executable like command-line tool or an application, or a system library with crate-type of staticlib or cdylib, check Cargo.lock into git. If you're curious about why that is, see "Why do binaries have Cargo.lock in version control, but not libraries?" in the FAQ.

https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

Reply:
OK, let's check it in then.